client/landing/subscriptions/components/site-list/site-row.tsx

```diff
@@ -107,16 +116,15 @@ export default function SiteRow( {
 			</span>
 			{ isLoggedIn && (
 				<span className="new-posts" role="cell">
-					{ newPostDelivery }
+					<SelectedNewPostDeliveryMethods
+						isEmailMeNewPostsSelected={ !! delivery_methods.email?.send_posts }
+						isNotifyMeOfNewPostsSelected={ !! delivery_methods.notification?.send_posts }
+					/>
 				</span>
 			) }
 			{ isLoggedIn && (
 				<span className="new-comments" role="cell">
-					{ delivery_methods?.email?.send_comments ? (
-						<Gridicon icon="checkmark" size={ 16 } className="green" />
-					) : (
-						<Gridicon icon="cross" size={ 16 } className="red" />
-					) }
+					{ delivery_methods.email?.send_comments ? <GreenCheck /> : <RedCross /> }
```

Comment:
Is it worth adding a tooltip to red cross to explain what it means?

Reply:
Good point. Created an issue https://github.com/Automattic/wp-calypso/issues/77488